docs/resources/r-and-d/yborrow.md

```diff
@@ -4,6 +4,4 @@ description: Not recommended for retail use
 
 # yBorrow
 
-{% embed url="https://yborrow.finance" caption="" %}
-
-It is an internal tool enabling credit delegation for smart contract to smart contract lending.
+[yBorrow](https://yborrow.finance) is an internal tool enabling credit delegation for smart contract to smart contract lending.
```

Comment:
Please remove yBorrow. That has evolved into the IronBank

Reply:
nice to know! thanks, will fix